mm/2s2h/BenGui/SearchableMenuItems.h

```diff
@@ -967,6 +967,12 @@ void AddEnhancements() {
                 WIDGET_CVAR_CHECKBOX,
                 {},
                 [](widgetInfo& info) { RegisterMoonJumpOnL(); } },
+              { "Elegy of Emptiness Anywhere",
+                "gCheats.ElegyAnywhere",
+                "Allows Elegy of Emptiness outside of Ikana",
+                WIDGET_CVAR_CHECKBOX,
+                {},
+                [](widgetInfo& info) { RegisterMoonJumpOnL(); } },
```

Comment:
Might need to re-run prettier after this
```suggestion
                {} },
```

Reply:
Whoops. Removed.